homework-1/src/templates/tips.html

```diff
@@ -0,0 +1,107 @@
+            <i class="fas fa-arrows-alt tips__second-left-icon"></i>
+        </div>
+        <div class="tips__second-right">
+            <div class="tips__area-icon-minus" id="tipsMinusThree">
+                <p class="tips__icon-minus">-</p>
+            </div>
+            <p class="tips__second-right-first-text">There’s nothing hackers love more than a dismissed update reminder. We keep our team’s computers in shape and up to date so that they aren’t subject to attacks that could otherwise be avoided.</p>
+            <p class="tips__second-right-second-text">Not properly updating your systems makes you an easy to reach target and hackers will often go after the lowest hanging fruit. When it comes to maintaining a strong cybersecurity network the best thing a user can do is to prevent problems before they occur, and the easiest way to do that is by making sure that all systems are updated and updated regularly. </p>
+            <div class="tips__second-right-i-area">
+                <img src="/assets/images/icon-i.png" alt="" class="tips__second-right-i">
+                <p class="tips__second-right-i-text">Based on the 2018 Verizon Data Breach Report</p>
+            </div>
+        </div>
+    </div>
+    <!-- 4 -->
+    <div class="tips__first" id="tipsFirstFour">
+        <p class="tips__first-one-letter">4</p>
+        <p class="tips__first-text">Secure Credentials</p>
+        <div class="tips__area-icon-plus" id="tipsPlusFour">
+            <p class="tips__icon-plus">+</p>
+        </div>
+     </div>
+     <div class="tips__second" id="tipsSecondFour">
+        <div class="tips__second-left">
+            <p class="tips__second-left-title">Secure Credentials</p>
+            <i class="fas fa-anchor tips__second-left-icon"></i>
+        </div>
+        <div class="tips__second-right">
+            <div class="tips__area-icon-minus" id="tipsMinusFour">
+                <p class="tips__icon-minus">-</p>
+            </div>
+            <p class="tips__second-right-first-text">There’s nothing hackers love more than a dismissed update reminder. We keep our team’s computers in shape and up to date so that they aren’t subject to attacks that could otherwise be avoided.</p>
+            <p class="tips__second-right-second-text">Not properly updating your systems makes you an easy to reach target and hackers will often go after the lowest hanging fruit. When it comes to maintaining a strong cybersecurity network the best thing a user can do is to prevent problems before they occur, and the easiest way to do that is by making sure that all systems are updated and updated regularly. </p>
+            <div class="tips__second-right-i-area">
+                <img src="/assets/images/icon-i.png" alt="" class="tips__second-right-i">
+                <p class="tips__second-right-i-text">Based on the 2018 Verizon Data Breach Report</p>
+            </div>
+        </div>
+    </div>
+
+    </div> <!--section -->
```

Comment:
actually it is good idea to use `<section>` tag for sections, but you use `<div>`

`<!--section -->` why do you have it here?

Reply:
<!--section --> delete. done